crates/conformance-test-utils/src/madara.rs

```diff
@@ -0,0 +1,85 @@
+use eyre::Result;
+use serde_json::Value;
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_storage_var_address;
+
+use crate::Felt;
+
+pub const FEE_TOKEN_ADDRESS_HEX: &str = "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7";
```

Comment:
isn't this already defined somewhere else? otherwise, can we put it in a `constants.rs` that we probably have already? or create one?

Reply:
Ah indeed there is this already [here](https://github.com/kkrt-labs/kakarot-rpc/blob/ad1435a037ccab9face468cffca28866187d02fc/crates/core/src/client/constants.rs#L12), thanks for spotting! I'll use this one instead